src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -565,7 +566,9 @@
             "withDependenciesTooltip": "Adds an external Apache commons-lang dependency to the generated project as an example how Maven dependencies should be used with Artemis exercises.",
             "recordTestwiseCoverage": "Record Testwise Coverage",
             "recordTestwiseCoverageTooltip": "Activate this option to record the testwise coverage for the solution repository. This option is only available for Java/Kotlin-exercises with non-sequential test runs.",
-            "customizeBuildPlansWithAeolusTooltip": "Activate this option to customize the build plans with Aeolus. This option is only available during exercise creation.",
+            "customizeBuildPlansWithAeolusTooltip": "Activate this option to customize the build plans with Aeolus.",
+            "customizeBuildPlansTooltip": "Activate this option to customize the build plan for this exercise.",
+            "revertToTemplateBuildPlan": "If you are not happy with your build plan, you can reset it to the template of the build plan by using the Reset button.",
```

Comment:
check if the german and english translations work.

Reply:
they work, I moved the entry so that it's the same for english and german